### DISCUSSION TITLE
Old `csrf_token` may be used after `$this->redirect('/dashboard', navigate: true);`

### DISCUSSION THREAD

**Discussion by nunomaduro** [livewire/livewire#6784] [7. Bugs (archived)] [answered]
### Livewire version

v3.0.3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Login user, and redirect using `navigate`.

```
Auth::attempt(...);

request()->session()->regenerate();

$this->redirect('/dashboard', navigate: true);
```

2. Any Livewire component update here will trigger 419, because it will still use the old csrf token.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by calebporzio** [comment, marked as answer]
Thanks for the report. Looks like Livewire was caching the initial pages CSRF token, so when it changed it wasn't picking it up. This has been fixed and will be in the latest release.

**Comment by emmadesilva** [comment]
Snyk seems to think this is serious. Maybe a security advisory would be prudent to add? https://security.snyk.io/vuln/SNYK-PHP-LIVEWIRELIVEWIRE-6219993
